_i18n/de/resources/moneropedia/node.md

```diff
@@ -1,24 +1,24 @@
 ---
-terms: ["node", "nodes", "full-node", "full-nodes"]
-summary: "a device on the Internet running the Monero software, with a full copy of the Monero blockchain, actively assisting the Monero network"
+terms: ["node", "nodes", "full-node", "full-nodes", "Node", "Nodes", "Full-Node", "Full-Nodes"]
+summary: "Ein mit dem Internet verbundenes Gerät mit einer Vollkopie der Monero-Blockchain, welches die Monero-Software ausführt und das Monero-Netzwerk aktiv unterstützt"
 ---
 
-{% include disclaimer.html translated="no" translationOutdated="no" %}
-### The Basics
+{% include disclaimer.html translated="yes" translationOutdated="no" %}
+### Grundlagen
 
-A device on the Internet running the Monero software, with a full copy of the Monero blockchain, actively assisting the Monero network.
+Ein Node ist ein mit dem Internet verbundenes Gerät mit einer Vollkopie der Monero-Blockchain, welches die Monero-Software ausführt und das Monero-Netzwerk aktiv unterstützt. Ein Node, der nicht auf deinem lokalen Gerät läuft, nennt sich @Remote-Node. Remote-Nodes können privat (bei ausschließlichem Privatgebrauch) oder offen sein, wenn sie durch andere zugänglich sind.
 
-### More Information
+### Weitere Informationen
 
-Nodes participate in the Monero network and secure @transactions by enforcing the rules of the network. Nodes download the entire @blockchain to know what transactions have taken place. Nodes assist the network by relaying transactions to other nodes on the network. Nodes may also choose to contribute to the Monero network by participating in crafting @blocks (this is called @mining).
+Nodes beteiligen sich innerhalb des Monero-Netzwerks, setzen dessen Regeln durch und sichern damit @Transaktionen auf diesem Netzwerk. Nodes laden die gesamte @Blockchain herunter, damit sie Kenntnis über die stattgefundenen Transaktionen haben. Sie unterstützen das Netzwerk, indem sie Transaktionen an andere Nodes innerhalb des Netzwerks weitergeben. Nodes können außerdem zum Monero-Netzwerk beitragen, indem sie an der Erstellung von @Blöcken teilhaben (dies wird @Mining genannt).
 
-Mining is the process by which nodes create a block from the previously accepted block, transactions that are waiting to be processed in the transaction pool, and the @coinbase-transaction. When a node believes it has crafted a valid block it will transmit the completed block to other nodes on the network and those nodes signal agreement by working on the next block in the chain.
+Mining ist der Prozess, in welchem Nodes ab dem Zeitpunkt, zu dem ein vorheriger Block akzeptiert wurde, neue Blöcke erstellen, während Transaktionen im Transaktionspool darauf warten, verarbeitet zu werden; zudem ist auch eine @Coinbase-Transaktion Teil des Minings. Wenn ein Node denkt, er habe einen validen Block erstellt, überträgt er diesen an andere Nodes innerhalb des Netzwerks. Diese wiederum signalisieren ihr Einverständnis durch das Arbeiten am nächsten Block der Blockchain.
```

Comment:
> Mining ist der Prozess, in welchem Nodes ab dem Zeitpunkt, zu dem ein vorheriger Block akzeptiert wurde, neue Blöcke erstellen, während Transaktionen im Transaktionspool darauf warten, verarbeitet zu werden; zudem ist auch eine @Coinbase-Transaktion Teil des Minings.

I think the meaning of the English text is different. The English version tries to explains what "content" blocks consist of:

> Mining is the process by which nodes create a block from 1) the previously accepted block (by referencing its hash), 2) transactions from the transaction pool, and 3) the @coinbase-transaction.

Reply:
> Mining ist der Prozess, in welchem Nodes unter Bezugnahme des Hashwerts des vorherigen Blocks aus den Transaktionen, die im Transaktionspool darauf warten, verarbeitet zu werden, und aus der @Coinbase-Transaktion einen neuen Block erstellen.

Would this translation work?

Reply:
```suggestion
Mining ist der Prozess, durch welchen Nodes unter Bezugnahme auf den vorherigen Block (meist mittels seines Hashwerts), aus den Transaktionen, die im Transaktionspool auf ihre Verarbeitung warten, und aus der @Coinbase-Transaktion einen neuen Block erstellen.
```

I put "meist mittels seines Hashwerts" between brackets because it doesn't exist in the English version.. if you want, we could also remove it.. but it might leave some readers wondering how the previous block is referenced. I placed a comma behind "Block," and shortened the part about transactions in the pool a bit.

I'm not sure about "der Prozess, in welchem..." - the "in" doesn't feel right. It reads better for me with "der Prozess, durch welchen...".

Reply:
For some reason I must have overseen your suggestion. I changed this part accordingly.